foundation-extensions/mobile-core/platform-sercvies/system-device-info-service.md

```diff
@@ -0,0 +1,34 @@
+## SystemInfo Service (iOS) & DeviceInfo Service (Android)
+
+## Overview
+
+The `SystemInfo Service` (iOS) and the `DeviceInfo Service`(Android) let you access critical pieces of information related to the user's device, such as carrier name, device name, locale, and more. 
```

Comment:
The `SystemInfo Service` (iOS) / `DeviceInfo Service`(Android) provides a set of APIs for retrieving device information, such as ...